spec/features/collection_multi_membership_spec.rb

```diff
@@ -95,7 +95,7 @@
           end
           # forwards to collections index page and shows flash message
           expect(page).to have_link 'All Collections'
-          expect(page).to have_link 'Your Collections'
+          expect(page).to have_link 'Collections'
```

Comment:
Switch this to...
```
    within('section.tabs-row') do
      expect(page).to have_link 'All Collections'
      expect(page).to have_link 'Your Collections'
    end
```

To be sure the tabs are correct, this change should probably be made for all the checks on the tab.